### ISSUE TITLE
witam mam pytanie co do Api Wysyłam z Allegro. Możemy zlecac odbiory za pośrednictwem Api ale czy możemy zlecac te odbiory z dwóch różnych adresów. Mamy dwa punkty odbioru z których kurierzy odbieraja od nas paczki.

### ISSUE THREAD

**Issue by trefllap** [allegro/allegro-api#4419]

**Comment by PrzemyslawLukanowski** [comment]
Tak, gdy tworzysz przesyłkę za pomocą [PUT /parcel-management/parcel-create-commands/{commandId}](https://developer.allegro.pl/documentation/#operation/createNewParcel), przekaż odpowiedni adres odbioru w polu `pickup.address.` Więcej szczegółów znajdziesz w [naszym poradniku](https://developer.allegro.pl/wza/#jak-utworzy%c4%87-now%c4%85-paczk%c4%99).
